forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt

```diff
@@ -0,0 +1,7 @@
+Name:You Are Unworthy of Mercy
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigSacEach | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, each opponent sacrifices a nonland permanent. If you control six or more lands, each opponent sacrifices three nonland permanents instead.
+SVar:TrigSacEach:DB$ Sacrifice | Amount$ X | Defined$ Player.Opponent | SacValid$ Permanent.nonLand | SacMessage$ nonland permanent
+SVar:X:Count$Compare Count$Valid Land.YouCtrl GE6.3.1

```

Comment:
was this tested?

Reply:
Sorry, I didn't have much time today :(

Reply:
no problem, no need to rush yourself

Reply:
any progress? :)

Reply:
OK, what exactly is the problem with this?

Reply:
I'm sceptical that the syntax works, that's why I was asking

Reply:
How about this? This seems how it is usually written (for example Frost Bite).

Reply:
Well checking ingame first would have been fine too lol but let's just move on :p